src/App.tsx

```diff
@@ -1,19 +1,72 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext, useEffect, useRef, useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Erorr } from './components/Erorr/Erorr';
+import { FilterTodoList } from './components/FilterTodoList/FilterTodoList';
+import { TodoList } from './components/TodoList/TodoList';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import { TypeError } from './types/ErorrType';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { FilterType } from './types/FilterType';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
try ti fix this rule and remove comment. Remove unused variables or use it
![image](https://user-images.githubusercontent.com/70009205/208136124-4e5664ce-a2fd-457a-9906-deac508c0b8b.png)